doc/sphinx/source/howto/operator.rst

```diff
@@ -0,0 +1,26 @@
+.. operator.rst
+
+############################
+Build a graph with operators
+############################
+
+This section illustrates the use of C++ operators to simplify the
+building of graphs.
+
+Several C++ operators have been overloaded so that when given nodes as
+arguments they construct a node that performs the operation on the
```

Comment:
Several C++ operators have been overloaded such that when given nodes as arguments, they may construct a simplified node to perform the operation / that performs the operation ...